lib/src/ovis_util/util.c

```diff
@@ -990,21 +990,53 @@ const char* ovis_errno_abbvr(int e)
 	return "UNKNOWN_ERRNO";
 }
 
+/** should be bigger than any expected strerror string.
+ * In 2020 ~50 was the upper limit.
+ */
+#define OVIS_SYS_ERRLEN 80
+
+/** should be bigger than any expected errno value.
+ * In 2020 ~140 was the upper limit.
+ */
+#define OVIS_SYS_NERR 256
+
+static char ovis_sys_errlist[OVIS_SYS_NERR][OVIS_SYS_ERRLEN];
+static int ovis_sys_errlist_initialized;
+static int ovis_sys_nerr;
+/* glibc has dropped support for sys_errlist.
+ * we copy it via strerror on first call.
+ * Make an unused call to ovis_strerror from main to avoid races.
```

Comment:
I think I would prefer to see an explicit initializer exported and employed. Have you followed this instruction everywhere that ovis_strerror is employed?

Reply:
added exported init function. it's use is optional.